### PR TITLE
Feature/update info

### PR DESCRIPTION
**Pull request by hixi**

### CODE REVIEW

File: Dockerfile.mediator [hunk 1]
```diff
@@ -8,9 +9,10 @@ ENV DJANGO_OSMAXX_CONVERSION_SERVICE_USERNAME=default_user DJANGO_OSMAXX_CONVERS
 ENV NUM_WORKERS=5 DATABASE_HOST=mediatordatabase DATABASE_PORT=5432 APP_PORT=8901 APP_HOST=0.0.0.0
 
 MAINTAINER HSR Geometalab <geometalab@hsr.ch>
+ENV DEBIAN_FRONTEND=noninteractive
```
Comment (das-g) [inline Dockerfile.mediator]: Should we reset this at the end of the Dockerfile? (I think it'll retain the value even into the running containers, if we don't. That might be unexpected when using apt in such a  container interactively, e.g. for debugging.)

File: Dockerfile.worker [hunk 2]
```diff
@@ -1,143 +1,110 @@
-# this gdal image comes with support for FileGDB and has python3.4 already installed
-FROM geometalab/gdal-debian:latest
+# this gdal image comes with support for FileGDB and has python3.6 already installed
+# is derived from ubuntu
```
Comment (das-g) [inline Dockerfile.worker]: This makes the comment non-grammatical.

File: Dockerfile.worker [hunk 3]
```diff
@@ -1,143 +1,110 @@
...
+FROM geometalab/gdal-docker:v3.0.0
 
 USER root
 
-ENV PYTHONUNBUFFERED=non-empty-string PYTHONIOENCODING=utf-8 LC_ALL=C.UTF-8 LANG=C.UTF-8
+ENV PYTHONUNBUFFERED=non-empty-string PYTHONIOENCODING=utf-8 LC_ALL=C.UTF-8 LANG=C.UTF-8 DEBIAN_FRONTEND=noninteractive
```
Comment (das-g) [inline Dockerfile.worker]: Should we reset `DEBIAN_FRONTEND` at the end of the Dockerfile? (I think it'll retain the value even into the running containers, if we don't. That might be unexpected when using apt in such a container interactively, e.g. for debugging.)

File: docs/development/project-development-environment.md [hunk 4]
```diff
@@ -133,7 +133,7 @@ ip route get 1 | awk '{print $NF;exit}'
 
 You can generate the complete URL in `sh` with:
 ```bash
-echo "http://$(ip route get 1 | awk '{print $NF;exit}'):8888"
+echo "http://$(ip route get 1 | awk '{print $NF;exit}'):8889"
```
Comment (das-g) [inline docs/development/project-development-environment.md]: Müsste das `awk`-Kommando hier auch angepasst werden?

File: tests/conftest.py [hunk 5]
```diff
@@ -205,6 +207,24 @@ def user(db, django_user_model, django_username_field):
     return user
 
 
+def create_authenticated_client(client, user):
```
Comment (das-g) [inline tests/conftest.py]: fixtures can AFAIR also be called like normal functions, so why not just inline this in `authenticated_client` below?

File: Dockerfile.frontend [hunk 6]
```diff
@@ -1,5 +1,6 @@
-# this gdal image comes with support for FileGDB and has python3.4 already installed
-FROM geometalab/gdal-debian:latest
+# this gdal image comes with support for FileGDB and has python3.6 already installed
+# is dervied from ubuntu
```
[the remark this answers is not in the document]
Comment (hixi) [reply Dockerfile.frontend]: I read them as two separate comments, has spelling errors anyway.

File: docs/development/project-development-environment.md (hunk 4 above)
Comment (hixi) [reply docs/development/project-development-environment.md]: Yes, missed that one, thanks.

File: tests/conftest.py (hunk 5 above)
Comment (hixi) [reply tests/conftest.py]: Nope, not anymore. That behaviour changed in pytest versions, don't know where exactly; that's actually the only reason I changed this.

File: Dockerfile.mediator (hunk 1 above)
Comment (hixi) [reply Dockerfile.mediator]: Good point.

File: Dockerfile.frontend [hunk 7]
```diff
@@ -1,13 +1,17 @@
-# this gdal image comes with support for FileGDB and has python3.4 already installed
-FROM geometalab/gdal-debian:latest
+# This gdal image comes with support for FileGDB and has python3.6 already installed.
```
Comment (das-g) [inline Dockerfile.frontend]:
```suggestion
# This GDAL image comes with support for FileGDB and has Python 3.6 already installed.
```

File: Dockerfile.frontend [hunk 8]
```diff
@@ -1,13 +1,17 @@
...
+# Based ubuntu.
```
Comment (das-g) [inline Dockerfile.frontend]:
```suggestion
# Based on image _/ubuntu.
```
or
```suggestion
# Based on official Ubuntu docker image.
```

File: Dockerfile.mediator [hunk 9]
```diff
@@ -1,5 +1,7 @@
-# this gdal image comes with support for FileGDB and has python3.4 already installed
-FROM geometalab/gdal-debian:latest
+# This gdal image comes with support for FileGDB and has python3.6 already installed.
```
Comment (das-g) [inline Dockerfile.mediator]:
```suggestion
# This GDAL image comes with support for FileGDB and has Python 3.6 already installed.
```

File: Dockerfile.mediator [hunk 10]
```diff
@@ -1,5 +1,7 @@
...
+# Based ubuntu.
```
Comment (das-g) [inline Dockerfile.mediator]:
```suggestion
# Based on image _/ubuntu.
```
or
```suggestion
# Based on official Ubuntu docker image.
```

File: Dockerfile.worker [hunk 11]
```diff
@@ -1,143 +1,114 @@
-# this gdal image comes with support for FileGDB and has python3.4 already installed
-FROM geometalab/gdal-debian:latest
+# This gdal image comes with support for FileGDB and has python3.6 already installed.
```
Comment (das-g) [inline Dockerfile.worker]:
```suggestion
# This GDAL image comes with support for FileGDB and has Python 3.6 already installed.
```

File: Dockerfile.worker [hunk 12]
```diff
@@ -1,143 +1,114 @@
...
+# Based ubuntu.
```
Comment (das-g) [inline Dockerfile.worker]:
```suggestion
# Based on image _/ubuntu.
```
or
```suggestion
# Based on official Ubuntu docker image.
```

**Pull request merged**